src/FixerFactory.php

```diff
@@ -79,9 +79,7 @@ public function setWhitespacesConfig(WhitespacesFixerConfig $config)
      */
     public function getFixers()
     {
-        $this->sortFixers();
-
-        return $this->fixers;
+        return Utils::sortFixers($this->fixers);
```

Comment:
originally, sorted fixers were assigned to `this->fixers`, so if one call method again, sorting will be ultra fast (as already sorted)

Reply:
The (now removed) `sortFixers()` method didn't seem to have a mechanism to check whether fixers are already sorted so I'm not sure this is true and I think it should be done in a separate PR.

Reply:
I'm not talking about any extra mechanism on our side.
PHP native `sort` is using quicksort alghoritm.
Complexity of sort operation depends on order of data, if they are in reverse order, complexity is `n*n`, if they are already in proper order, complexity is `n * log(n)`. 

so, nothing to do here on our side, except remembering the sorted order
```diff
-return Utils::sortFixers($this->fixers);
+$this->fixers = Utils::sortFixers($this->fixers);
+return $this->fixers;
```

Reply:
Done.